src/main/java/chess/GameCommand.java

```diff
@@ -0,0 +1,88 @@
+package chess;
+
+import chess.domain.board.File;
+import chess.domain.board.Rank;
+import chess.domain.board.Square;
+import java.util.List;
+
+public class GameCommand {
+    private enum gameCommandIndex {
+        COMMAND(0),
+        SOURCE(1),
+        TARGET(2);
+
+        private final int index;
+
+        gameCommandIndex(final int index) {
+            this.index = index;
+        }
+    }
+
+    private enum moveIndex {
```

Comment:
```suggestion
    private enum MoveIndex {
```

클래스이기 때문에 대문자로 시작해야할 것 같아요.

Reply:
자바 네이밍 규칙에 대해서 다시 한 번 정리하고 전체적으로 수정하였습니다 ! 🥺